lib/viewing/ViewsFromServer.js

```diff
@@ -22,7 +22,11 @@ var ViewsFromServer = {
     },
 
     get: function(uid) {
-        return viewElements[uid] || null;
+        var viewElement = viewElements[uid] || null;
+
+        viewElements[uid] = null;
```

Comment:
Can you do `delete viewElements[uid];` to release references for garbage collection. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete